### PR TITLE
added random content function to button

### PR DESCRIPTION
**Pull request by gralmurdok**

### CODE REVIEW

File: src/actions/neuronActions.js [hunk 1]
```diff
@@ -12,8 +12,14 @@ const setCurrentContent = content => ({
...
 });
 
+const setCurrentRecomendedContents = content => ({
+  type: actionTypes.LOAD_RECOMENDED_CONTENTS,
+  payload: content,
+});
+
 const loadNeuronByIdAsync = id => async (dispatch) => {
   const res = await api.neuron.getNeuronById(id);
+  console.log(res.data);
```
Comment (leninkp) [inline src/actions/neuronActions.js]: console log :macool:

File: src/constants.js [hunk 2]
```diff
@@ -2,4 +2,4 @@ export const LANDSCAPE = 'LANDSCAPE';
 export const PORTRAIT = 'PORTRAIT';
 export const DRAWER_OFFSET = 220;
 export const FLORECIDA = 'florecida';
-export const URL_BASE = __DEV__ ? 'http://localhost:5000' : '';
\ No newline at end of file
+export const URL_BASE = __DEV__ ? 'http://moi-staging.herokuapp.com' : '';
```
Comment (leninkp) [inline src/constants.js]: :wat:

File: src/containers/RandomContents/RandomContents.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  componentDidMount() {
+    this.getRandomContents();
+  }
+
+  getRandomContents = async () => {
+    const { loadRecomendedContents, loadNeuronByIdAsync } = this.props;
+    const recomendedContents = await loadRecomendedContents(1);
+    const neurons = (recomendedContents.neurons || []);
+    const randomNeuron = (neurons[Math.floor(Math.random() * neurons.length)] || {});
+    console.log(randomNeuron);
```
Comment (leninkp) [inline src/containers/RandomContents/RandomContents.js]: remove log

Comment (daosgc) [reply src/containers/RandomContents/RandomContents.js]: console.log('cargando in production');

**Pull request merged**